packages/e2e/docker-compose.yml

```diff
@@ -36,6 +36,14 @@ services:
       options:
         max-size: "200k"
         max-file: "10"
+  files-server:
+    image: nginx:${NGINX_VERSION:-1.22.1-alpine}
+    volumes:
+      - ./local-network/files-server:/usr/share/nginx/html
+    ports:
+      - "8080:80"
```

Comment:
```suggestion
      - "${FILE_SERVER_PORT:-8080}:80"
```

Reply:
It will currently clash with any existing services on the host bound to 8080 without a way to provide a custom mapping.

Reply:
> you will have have to change this url: http://files-server:NEW_PORT/IOG1.json

No, `ports` is just the host mapping, and since you're referring to the service by name, that script must be running within the Docker network which is not impacted.